OpenSim/Actuators/Millard2012EquilibriumMuscle.h

```diff
@@ -320,6 +332,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Millard2012EquilibriumMuscle, Muscle);
     @returns The time derivative of activation. */
     double getActivationDerivative(const SimTK::State& s) const;
 
+    /** get the passive elastic force in the fiber (N) */
+    double getPassiveFiberElasticForce(const SimTK::State& s) const;
+    /** get the passive elastic force in the fiber (N) projected onto the tendon
+        direction */
+    double getPassiveFiberElasticForceAlongTendon(const SimTK::State& s) const;
+    /** get the passive damping force in the fiber (N) */
+    double getPassiveFiberDampingForce(const SimTK::State& s) const;
+    /** get the passive damping force in the fiber (N) projected onto the tendon
+        direction */
+    double getPassiveFiberDampingForceAlongTendon(const SimTK::State& s) const;
```

Comment:
If I didn't have additional background, these doxygen comments probably would not be sufficient for me. What about:

```
get the portion of the passive fiber force coming from the damping element only
```

and similar for the elastic element.